Form/TypeExtension/MultiUploadCollectionExtension.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace FSi\Bundle\FormExtensionsBundle\Form\TypeExtension;
+
+use FSi\Bundle\FormExtensionsBundle\Form\EventListener\MultiUploadCollectionListener;
+use Symfony\Component\Form\AbstractTypeExtension;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+
+class MultiUploadCollectionExtension extends AbstractTypeExtension
+{
+    /**
+     * @inheritdoc
+     */
+    public function getExtendedType()
+    {
+        return 'collection';
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function setDefaultOptions(OptionsResolverInterface $resolver)
```

Comment:
you should add `configureOptions` method for compatibility with future version of symfony